res/controllers/midi-components-0.0.js

```diff
@@ -57,7 +57,12 @@
         // default attributes
         // You should probably overwrite at least some of these.
         inValueScale: function (value) {
-            return value / this.max;
+            // Hack to get exact center of pots to return 0.5
+            if (value > (this.max / 2)) {
+                return (value - 1) / (this.max - 1);
+            } else {
+                return value / (this.max + 1);
+            }
```

Comment:
There is no such reverse function, but that may be helpful for encoders with LED rings like those on the Electrix Tweaker and MIDI Fighter Twister.